designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/StatisticsUrlsDomainService.scala

```diff
@@ -0,0 +1,89 @@
+package pl.touk.nussknacker.ui.statistics
+
+import cats.implicits.toTraverseOps
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.ui.statistics.StatisticsUrlsDomainService.{
+  emptyString,
+  nuStatsUrl,
+  queryParamsSeparator,
+  toURL,
+  urlBytesSizeLimit
+}
+
+import java.net.{URI, URL, URLEncoder}
+import java.nio.charset.StandardCharsets
+import scala.collection.mutable.ArrayBuffer
+import scala.util.{Failure, Success, Try}
+
+class StatisticsUrlsDomainService(private val fingerprint: Fingerprint, private val statistics: Map[String, String])
```

Comment:
WDYT about renaming it to Statistics? And this class could be a case class.

The `statistics` field could renamed to `rawStattistics` or sth like that